### PR TITLE
Fb login

### PR DESCRIPTION
**Pull request by superzzp**

### CODE REVIEW

File: src/components/Authentication/Login.js [hunk 1]
```diff
@@ -122,7 +128,22 @@ function Login(props) {
               Sign In with Google
             </Button>
           </CardContent>
-          <Divider variant="middle" />
+          <CardContent>
```
Comment (ianmah) [inline src/components/Authentication/Login.js:131]: I believe we can put the facebook button inside the same cardcontent component as the google button?

File: src/components/Authentication/LoginRedirect.js [hunk 2]
```diff
@@ -18,6 +18,7 @@ export class LoginRedirect extends Component {
     pollForAuthenticatedUser() {
         Auth.currentAuthenticatedUser()
             .then(user => {
+                console.log('USER: ===> ', user);
```
Comment (ianmah) [inline src/components/Authentication/LoginRedirect.js:21]: 🔪

File: src/utils/index.js [hunk 3]
```diff
@@ -1,8 +1,9 @@
 import aws_config from '../aws-config';
-
-export const API_URL = process.env.REACT_APP_STAGE  === 'production'
-    ? process.env.REACT_APP_PROD_API
-    : process.env.REACT_APP_STAGING_API
+//COMMENT OUT FOR TESTING, USING STAGING NOW
```
Comment (ianmah) [inline src/utils/index.js:2]: The code that you commented out will always use the staging api when developing locally

File: src/components/Authentication/Login.js [hunk 4]
```diff
@@ -72,6 +77,7 @@ function Login(props) {
   const classes = useStyles();
   Auth.currentAuthenticatedUser()
     .then(user => {
+      console.log(user);
```
Comment (ianmah) [inline src/components/Authentication/Login.js:80]: 🔪

File: src/components/Authentication/Login.js [hunk 5]
```diff
@@ -25,6 +25,11 @@ const styles = {
     marginTop: "5px",
     marginRight: "8px",
     width: "19px"
+  },
+  facebookIconBtn: {
+    backgroundColor: "#3fb5a3",
```
Comment (ianmah) [inline src/components/Authentication/Login.js:30]: Can we make the facebook backgrouind colour blue and the google background colour white?

File: src/components/Authentication/Login.js [hunk 6]
```diff
@@ -122,7 +128,22 @@ function Login(props) {
               Sign In with Google
             </Button>
           </CardContent>
-          <Divider variant="middle" />
+          <CardContent>
+            <Button style={styles.facebookIconBtn}
```
Comment (ianmah) [inline src/components/Authentication/Login.js:132]: Instead of adding styles can we use the color prop?

Comment (superzzp) [reply src/components/Authentication/Login.js:136]: the styling for facebookIconBtn also includes margins tho

Comment (ianmah) [reply src/components/Authentication/Login.js:136]: Can we remove background color (on google and facebook) and use the color prop instead?

File: src/components/Authentication/Login.js (hunk 4 above)
Comment (dchen150) [reply src/components/Authentication/Login.js:85]: @superzzp can we clean up any remaining console.logs

File: src/components/Authentication/LoginRedirect.js (hunk 2 above)
Comment (superzzp) [reply src/components/Authentication/LoginRedirect.js]: Updated

File: src/utils/index.js (hunk 3 above)
Comment (superzzp) [reply src/utils/index.js]: Updated

File: src/components/Authentication/Login.js (hunk 4 above)
Comment (superzzp) [reply src/components/Authentication/Login.js]: updated

File: src/components/Authentication/Login.js (hunk 1 above)
Comment (superzzp) [reply src/components/Authentication/Login.js]: Sounds good, updated in the latest version

File: public/index.html [hunk 7]
```diff
@@ -13,9 +13,9 @@
       manifest.json provides metadata used when your web app is installed on a
       user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
     -->
-  <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+  <meta rel="manifest" href="%PUBLIC_URL%/manifest.json" />
```
Comment (ianmah) [inline public/index.html:16]: what is the change here for?

File: public/index.html [hunk 8]
```diff
@@ -13,9 +13,9 @@
...
   <!--
-      Notice the use of %PUBLIC_URL% in the tags above.
+      Nlinkotice the use of %PUBLIC_URL% in the tags above.
```
Comment (ianmah) [inline public/index.html:18]: undo

File: src/components/Authentication/Login.js (hunk 6 above)
Comment (superzzp) [reply src/components/Authentication/Login.js:122]: the color prop on buttons only supports the default color
https://material-ui.com/zh/api/button/

Comment (superzzp) [reply src/components/Authentication/Login.js:116]: updated, use the theme provider for the background color

File: src/components/Authentication/Login.js (hunk 5 above)
Comment (superzzp) [reply src/components/Authentication/Login.js]: updated

File: .vscode/launch.json [hunk 9]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (ianmah) [inline .vscode/launch.json:1]: what is this file and can we get rid of it?

Comment (superzzp) [reply .vscode/launch.json]: the file is removed in the latest commit

File: .gitignore [hunk 10]
```diff
@@ -42,4 +42,8 @@ awsconfiguration.json
 .idea/
 
 #firebase
-.firebase
\ No newline at end of file
+.firebase
+
+#vscode
+.vscode/launch.json
+.vscode/settings.json
```
Comment (ianmah) [inline .gitignore:49]: we need this file

Comment (superzzp) [reply .gitignore]: updated

**Pull request merged**